corehq/util/django_migrations.py

```diff
@@ -99,3 +100,10 @@ def _inner(*args, **kwargs):
             return
         return migration_fn(*args, **kwargs)
     return _inner
+
+
+def noop_migration():
+    """
+    A migration that does nothing. Used to replace old migrations that are no longer required e.g moved.
+    """
+    return RunPython(RunPython.noop, RunPython.noop)
```

Comment:
Should this be [`elidable=True`](https://docs.djangoproject.com/en/2.2/ref/migration-operations/#runpython)?

Reply:
yes

Reply:
At @emord 's suggestion, I removed nearly all `noop_migration`s, save a few which used this dynamically.  Those ones I think _don't_ want `elidable=True`, since whether the migration is actually a noop depends on the settings.  I changed the docstring to reflect the intended usage.